src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardHelper.js

```diff
@@ -89,6 +89,10 @@
      * @param allFields: parsed list of all possible fields for dueling picklist
      */
     loadAvailableFields: function(component, activeFields, allFields) {
+        const FIELD_SOBJECT_TYPE_OPPORTUNITY = 'opportunity';
```

Comment:
It might be worth putting a bit of logic in between line 145 ("var allFieldsBySObject = this.groupFieldsBySObject(everyField);") and 147 (where it loops the the sobject names in the keys of allFieldsBySObject) to make sure that opportunity and payment are the first and second keys (assuming they exist) and then whatever other sobjects might be in that map after that

Reply:
Agreed, will make that change.